root/models/base.py

```diff
@@ -25,16 +24,18 @@ def save(self, commit=True, *args, **kwargs):
         a save_m2m() method to the form which can be called after the instance
         is saved manually at a later time. Return the model instance.
         """
+        # pylint: disable=invalid-string-quote
+        # pylint: disable=W0212
+        # pylint: disable=C0209
+        # pylint: disable=W0201
```

Comment:
Kan du bruke navn i stedet for koden? Det er lettere å lese.

Reply:
Fikset ved å legge inn navn